client-mu-plugins/plugin-loader.php

```diff
@@ -11,5 +11,10 @@
  */
 
 // wpcom_vip_load_plugin( 'plugin-name' );
-// Note the above requires a specific naming structure: /plugin-name/plugin-name.php
-// You can also specify a specific root file: wpcom_vip_load_plugin( 'plugin-name/plugin.php' );
+/**
+ * Note the above requires a specific naming structure: /plugin-name/plugin-name.php
+ * You can also specify a specific root file: wpcom_vip_load_plugin( 'plugin-name/plugin.php' );
+ *
+ * wpcom_vip_load_plugin only loads plugins from the `WP_PLUGIN_DIR` directory.
+ * For client-mu-plugins `requre __DIR__ '/plugin-name/plugin-name.php'` works.
```

Comment:
```suggestion
 * For client-mu-plugins `require __DIR__ '/plugin-name/plugin-name.php'` works.
```

Reply:
@jenkoian I think we both missed the `.` ;)

`require __DIR__ . '/plugin-name/plugin-name.php'`